CHANGELOG.md

```diff
@@ -12,4 +12,4 @@ In this file, all the changes are listed by release.
 
 ## 100.10
 
-* Introduction of new toolkit architecture.
\ No newline at end of file
+* Introduction of new toolkit architecture.
```

Comment:
I cannot tell what actually changed on this line.

Reply:
Probably a missing newline?